pkgs/development/libraries/libtickit/default.nix

```diff
@@ -1,54 +1,67 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, nix-update-script
 , pkg-config
 , libtool
 , perl
 , libtermkey
 , unibilium
 }:
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libtickit";
   version = "0.4.3";
 
   src = fetchFromGitHub {
     owner = "leonerd";
     repo = "libtickit";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
     hash = "sha256-QCrym8g5J1qwsFpU/PB8zZIWdM3YzOySknISSbQE4Sc=";
   };
 
+  enableParallelBuilding = true;
+
+  #
+  # Both the build and the check phase need a libtool path
```

Comment:
We don't need the fences around the comment and also I don't think we need to comment this ata all.

Reply:
Removed the comment